### DISCUSSION TITLE
开发了一个115 cookie自动同步到alist的插件

### DISCUSSION THREAD

**Discussion by yenkn** [AlistGo/alist#7737] [Show and tell]
用115的qrcode/cookie方式登录总是会掉线，并且经常要登录网页端添加离线下载任务，每次登录原来的cookie就会失效，因此开发了一个自动将chrome的115 cookie同步到alist的插件。欢迎试用反馈

仓库地址：[https://github.com/yenkn/115-to-alist](https://github.com/yenkn/115-to-alist)

使用方法：
![115-to-alist tutorial](https://github.com/yenkn/115-to-alist/raw/main/docs/tutorial.png)

**Comment by KSAMNI** [comment]
有用的
